packages/colorpickers/stories/examples/ColorpickerDialog/WithIconButton.tsx

```diff
@@ -42,23 +43,25 @@ export const WithIconButton: Story = ({
     <Grid>
       <Row style={{ minHeight: 470 }}>
         <Col textAlign="center">
-          <ColorpickerDialog
-            color={color}
-            labels={labels}
-            onChange={setColor}
-            hasArrow={hasArrow}
-            placement={placement}
-            isAnimated={isAnimated}
-            onClose={setSelectedColor}
-          >
-            <IconButton
-              aria-label="palette"
-              disabled={disabled}
-              style={{ color: disabled ? undefined : iconColor }}
+          <Tooltip content="Color picker palette">
```

Comment:
So is this unable to directly wrap the `IconButton`? As in:

```js
<ColorpickerDialog>
  <Tooltip content="Color picker palette">
    <IconButton>
      <PaletteIcon />
    </IconButton>
  </Tooltip>
</ColorpickerDialog>
```

Or even using a `forwardRef` like:

```js
const PaletteIconButton = React.forwardRef(
  (props: React.ComponentPropsWithoutRef<'button'>, ref: React.Ref<HTMLButtonElement>) => (
    <Tooltip content="Palette">
      <IconButton aria-label="palette" ref={ref} {...props}>
        <PaletteIcon />
      </IconButton>
    </Tooltip>
  )
);

<ColorpickerDialog>
  <Tooltip content="Palette">
    <PaletteIconButton />
  </Tooltip>
</ColorpickerDialog>
```

Reply:
I think the `Tooltip` is clobbering the trigger element in the above use cases. I think we could get that to work, but may need to update the `Tooltip` modal package to ensure the proper refs are merged for Popper.

We had decided that it was best to move forward without making modifications to tooltips for now.

Reply:
Updated the Storybook example with using a `forwardRef` 👍🏽